src/gpio.rs

```diff
@@ -1385,6 +1541,8 @@ mod chip {
     use crate::adc::{ADC1, ADC2};
 
     use super::*;
+    #[cfg(feature = "nightly")]
```

Comment:
As per above, no need for `nightly`. Perhaps `experimental`, but I don't have strong feelings here. And !`riscv-ulp-hal`.